packages/react-server/src/ReactFizzServer.js

```diff
@@ -1585,15 +1621,36 @@ export function performWork(request: Request): void {
   const prevResponseState = currentResponseState;
   setCurrentResponseState(request.responseState);
   try {
-    const pingedTasks = request.pingedTasks;
-    let i;
-    for (i = 0; i < pingedTasks.length; i++) {
-      const task = pingedTasks[i];
-      retryTask(request, task);
-    }
-    pingedTasks.splice(0, i);
-    if (request.destination !== null) {
-      flushCompletedQueues(request, request.destination);
+    let queue;
```

Comment:
This is the main logic loop. Priority of work:

1. Pinged tasks (Boundaries that have completed)
2. Fallback tasks
3. Flushing  pinged and Fallback tasks
4. CPU Suspense tasks
5. Flushing Partial queues

Reply:
@sebmarkbage Do you still want to implement CPU Suspense for Fizz?

Reply:
Yea. I think maybe @mofeiZ was going to look into to taking on that.